content/_changelogs/10.5.0.md

```diff
@@ -0,0 +1,85 @@
+## 10.5.0
+
+_Released 8/15/2022_
+
+**Features:**
+
+- Display a 'flaky test' indicator in the Specs Explorer for specs that are
+  known to be flaky in the Dashboard. Addresses
+  [#22656](https://github.com/cypress-io/cypress/issues/22656) &
+  [#22657](https://github.com/cypress-io/cypress/issues/22657).
+- The project drop down is now populated when connecting a project with an
+  organization that only has one project. Addresses
+  [#22936](https://github.com/cypress-io/cypress/issues/22936).
+- Added new `testIsolation` configuration option to allow users to revert to
+  `legacy` mode when `experimentalSessionAndOrigin` is set to true. Read more
+  about [test isolation](/guides/core-concepts/test-isolation) in Cypress to
+  learn more. Addresses
+  [#22279](https://github.com/cypress-io/cypress/issues/22279).
+- React 18 projects will now be scaffolded with the `cypress/react18` mount
+  registration during component setup. Addresses
+  [#23033](https://github.com/cypress-io/cypress/issues/23033).
+- The list of Vue components in the project are now available for selection when
```

Comment:
We should note here that this is currently only an option for projects that don't have a `specPattern` configured for component testing. Supporting that is in PR https://github.com/cypress-io/cypress/pull/23298

Reply:
@astone123 Can you suggest the correct entry here

Reply:
> A list of Vue components in the project are now available for selection when generating a new spec for component testing in projects that are using the default spec pattern.

Reply:
updated [d374886](https://github.com/cypress-io/cypress-documentation/pull/4668/commits/d374886df30be68093496db775cc8e7f24ca1443)